GVFS/GVFS.Build/GVFS.props

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup Label="Parameters">
     <GVFSVersion>0.2.173.2</GVFSVersion>
-    <GitPackageVersion>2.20181018.1</GitPackageVersion>
+    <GitPackageVersion>2.20181107.1</GitPackageVersion>

```

Comment:
This change is incorrectly shown by GitHub, after I picked changes on master from upstream to my fork and rebased mac_installer_prototype branch on to master.

Use this [URL](https://github.com/Microsoft/VFSForGit/compare/master...alameenshah:mac_installer_prototype) to view the correct diff.